package.json

```diff
@@ -63,7 +63,9 @@
     "pdfjs-dist": "3.3.122",
     "rxjs": "6.6.6",
     "tslib": "^2.5.3",
-    "zone.js": "0.11.8"
+    "zone.js": "0.11.8",
+    "lottie-web": "^5.12.2",
+    "ngx-lottie": "^9.1.0"
```

Comment:
it it mandatory to use this library? it seems they also have some analytics:

Please note: ngx-lottie uses [Scarf](https://scarf.sh/) to collect anonymized installation analytics. These analytics help support the maintainers of this library. However, if you'd like to opt out, you can do so by setting scarfSettings.enabled = false in your project's package.json. Alternatively, you can set the environment variable SCARF_ANALYTICS=false before you install.

Reply:
I'm waiting for information about that. Will let you know when I get answer.

Reply:
after discussion with designers I have removed that library